service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/digitaltwin/authentication/BearerTokenProvider.java

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+package com.microsoft.azure.sdk.iot.service.digitaltwin.authentication;
+
+/**
+ * Interface for providing a Bearer authentication token for use in RBAC based authentication.
+ */
+public interface BearerTokenProvider {
```

Comment:
This new interface is basically the same as the existing ```SasTokenProvider``` but the token retrieval method in this interface specifies the token is a bearer token rather than a SAS token